modin/pandas/test/test_groupby.py

```diff
@@ -1247,18 +1248,25 @@ def test_shift_freq(groupby_axis, shift_axis):
             ],
             "agg_dict": {
                 "max": (list(test_data["int_data"].keys())[1], max),
-                "min": (list(test_data["int_data"].keys())[-1], min),
+                "min": (list(test_data["int_data"].keys())[-2], min),
```

Comment:
Why?

Reply:
It's to avoid the intersection of 'by' columns and columns to aggregate, which causes some problems in Modin when `as_index=False` (#2542). This test was kept below but still marked as `skip`.